desktop/electron/reportProcessing.ts

```diff
@@ -14,11 +14,43 @@ import topics from "./mockData/topics.json";
 import visuals from "./mockData/visuals.json";
 import voiceovers from "./mockData/voiceovers.json";
 
+interface AudioInfo {
+  audioUrl: string;
+  duration: number;
+  text: string;
+}
+
+// Takes in an array of strings and returns an array of AudioInfo
+export async function textToAudio(textArray: string[]): Promise<AudioInfo[]> {
+
+    const audioInfoArray: AudioInfo[] = [];
+
+    for (const text of textArray) {
+      const postData = new FormData();
+      postData.append('script', text);
+
+      const response = await fetch('http://0.0.0.0:8888/tts/', {

```

Comment:
This assumes the server is running on the user's local machine or the port is being port forwarded. Instead use https://iguana.alexo.uk/tts/